src/nextcloud/bin/nextcloud-cron

```diff
@@ -11,6 +11,11 @@ done
 printf "done\n"
 
 while true; do
-	run-php "$SNAP/htdocs/cron.php"
-	sleep 15m
+    # Exit the entire job if the requested interval is -1
+    if [ "$(cronjob_interval)" = "-1" ]; then
+        exit 0
+    fi
+
+    run-php "$SNAP/htdocs/cron.php"
+    sleep "$(cronjob_interval)"
```

Comment:
Looks good, but please leave these as tabs.

Reply:
I must have messed this up when copy pasting it. It's now fixed.